src/test/java/juja/sqlcmd/DatabaseManagerTest.java

```diff
@@ -129,23 +130,60 @@ public void getTableDataWithNotExistingTable() throws SQLException {
 
     @Test
     public void getTableDataWithValidTableTwoRows() throws SQLException {
-        executeSqlQuery("CREATE TABLE test_table(" +
-                "id INTEGER," +
-                "name VARCHAR(128)" +
-                ")");
-        executeSqlQuery("INSERT INTO test_table VALUES(1,'name1')");
-        executeSqlQuery("INSERT INTO test_table VALUES(2,'name2')");
+        createTestTableWithIdAndName(TEST_TABLE_NAME);
+        executeSqlQuery("INSERT INTO " + TEST_TABLE_NAME + " VALUES(1,'name1')");
+        executeSqlQuery("INSERT INTO " + TEST_TABLE_NAME + " VALUES(2,'name2')");
         DataSet row1 = new DataSet(2);
         row1.insertValue(0, "1");
         row1.insertValue(1, "name1");
         DataSet row2 = new DataSet(2);
         row2.insertValue(0, "2");
         row2.insertValue(1, "name2");
         DataSet[] expectedArray = new DataSet[]{row1, row2};
-        DataSet[] actualArray = databaseManager.getTableData("test_table");
+        DataSet[] actualArray = databaseManager.getTableData(TEST_TABLE_NAME);
         assertThat(actualArray, arrayContainingInAnyOrder(expectedArray));
     }
 
+    @Test
+    public void insertIntoTableWithExistingTable() throws SQLException {
+        createTestTableWithIdAndName(TEST_TABLE_NAME);
+        DataSet row = new DataSet(2);
+        row.insertValue(0, "1");
+        row.insertValue(1, "name1");
+        assertTrue(databaseManager.insert(TEST_TABLE_NAME, row));
+    }
+
+    @Test
+    public void insertIntoTableWithNotExistingTable() {
+        DataSet row = new DataSet(2);
+        row.insertValue(0, "1");
+        row.insertValue(1, "name1");
+        assertFalse(databaseManager.insert("tableDoesNotExist", row));
+    }
+
+    @Test
+    public void insertIntoTableWithExceedingNumberOfParameters() throws SQLException {
```

Comment:
Maybe insertWhenExtraParameters?

Reply:
May be. I'll use your name, but for me "extra" is something good, like a bonus))